deployment/helm/ditto/values.yaml

```diff
@@ -302,6 +302,14 @@ ingress:
     enabled: false
     # replicaCount configuration for the ingress controller
     replicaCount: 1
+    config:
+      workerProcesses: 2
```

Comment:
could you add a documentation line before, please?

Reply:
added necessary explanation